packages/shared/src/components/Feed.tsx

```diff
@@ -386,6 +389,9 @@ export default function Feed<T>({
     },
     post,
   };
+
+  const ArticleModal = PostModalMap[selectedPost?.type];
```

Comment:
I suggest `PostModal` for better naming, not blocking

Reply:
It is already used